controllers/content_items_controller.go

```diff
@@ -13,35 +15,59 @@ type ContentItemsController struct {
 	arbiter           *urlarbiter.URLArbiter
 	liveContentStore  *contentstore.ContentStoreClient
 	draftContentStore *contentstore.ContentStoreClient
+	errbitNotifier    *errornotifier.ErrbitNotifier
```

Comment:
It would be better for this to be a `errornotifier.Notifier` instead of a `*errornotifier.ErrbitNotifier`.  That way the controller doesn't need to know what sort of error notifier it's dealing with.